air/src/proof/security.rs

```diff
@@ -0,0 +1,728 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+//! Contains helper structs and methods to estimate the security of STARK proofs.
+
+use core::cmp;
+
+use crate::ProofOptions;
+
+const GRINDING_CONTRIBUTION_FLOOR: u32 = 80;
+const MAX_PROXIMITY_PARAMETER: u64 = 1000;
+
+pub struct ConjecturedSecurity(u32);
```

Comment:
nit: let's add a section separator above this line with title `CONJECTURED SECURITY`.

Also, let's add doc comments to this struct.

Reply:
added